### PR TITLE
[GLIV-251] fix Centos65 offline installter for StateGrid

### PR DESCRIPTION
**Pull request by Eric-Chu**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -283,7 +283,8 @@ if [[ ! $ONLINE && ! $OFFLINE ]]; then
 fi 
 
 if [ "Q$OS" = "QRHEL" ]; then
-  declare -a arr=("tar" "iputils" "wget")
+  #declare -a arr=("tar" "iputils" "wget")
```
Comment (justinligraph) [inline install.sh]: Why remove this? will online still work without wget?

File: install.sh [hunk 2]
```diff
@@ -283,7 +283,8 @@ if [[ ! $ONLINE && ! $OFFLINE ]]; then
...
+  declare -a arr=("tar")
   for i in "${arr[@]}"
   do
     if ! rpm -q "$i" >/dev/null 2>&1; then
```
Comment (justinligraph) [inline install.sh]: Will this need to go online get them? for offline, maybe check if the command exists, if not error out.

File: install.sh [hunk 3]
```diff
@@ -378,7 +383,9 @@ progress "Installing required system software packages ..."
 if [ "Q$OS" = "QRHEL" ]; then  # Redhat or CentOS
   yum remove -y graphsql
   if [ "$OFFLINE" = true ]; then
-    yum --disablerepo=* --enablerepo=GraphSQL-Local install -y ${pkg_name}
+    yum --disablerepo=* --enablerepo=graphsql-Local install -y ${pkg_name}
+    java_package=$(yum --disablerepo=* --enablerepo=graphsql-Local list | grep java | grep 1.7 | grep -v devel | cut -d ' ' -f1)
```
Comment (mingxiw) [inline install.sh]: please make sure you have test cases to can be run forever.

**Comment by Eric-Chu** [comment]
Hi Mingxi,

we don't have UT for the system prerequisites installation. But we do have
End-2-End test for the System Prerequisite Installation in our Jekins
Server. Thanks

File: install.sh (hunk 3 above)
Comment (gaolk) [inline install.sh]: @Eric-Chu  and @justinligraph  What 6.5 did we test? Do we get the State Grid’s 6.5 minor version from @xchang2014 ?

**Comment by Eric-Chu** [comment]
Hi like,

I tested on the Centos6.5 Minimal ISO version. It's from the official
website.

Eric

**Pull request merged**